src/main/java/Homework4/CarInterface.java

```diff
@@ -0,0 +1,9 @@
+package Homework4;
+
+interface CarInterface {
```

Comment:
Why do we need this extra interface?

Reply:
Initial am structurat diferit proiectul Vehicule. l-am adaptat insa dupa ce am vazut rezolvarea la cursul de miercuri.
Metodele din  CarInterface le-am mutat asa ca ea a ramas goala :)

Reply:
I forgot, I was supossed to reply in english